### DISCUSSION TITLE
Audit trail of configuration changes made to GitHub Project Boards

### DISCUSSION THREAD

**Discussion by RobertKielty** [community/community#65445] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

Is it possible to review the configuration changes made to a Project Board? 

I would like to be able to see 
Who made the change, when a change made, what the changes were.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
👋🏼 We don't have history of configuration changes made to a project - but are the events listed above the ones you're interested in seeing? If you have any more please let us know!
